GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -50,9 +50,10 @@ protected override void Execute(GVFSEnlistment enlistment)
                 this.WriteMessage(string.Empty);
                 this.WriteMessage("gvfs version " + ProcessHelper.GetCurrentProcessVersion());
 
-                GitVersion gitVersion;
-                string error;
-                if (GitProcess.TryGetVersion(out gitVersion, out error))
+                GitVersion gitVersion = null;
+                string error = null;
+                string gitPath = GVFSPlatform.Instance.GitInstallation.GetInstalledGitBinPath();

```

Comment:
This method is being called again a few lines later. Since you're already storing it in a variable, use the variable on line 65 too

Reply:
updated line 65 to read the git path from enlistment.